src/main/resources/hudson/plugins/descriptionsetter/DescriptionSetterBuilder/config.jelly

```diff
@@ -6,4 +6,7 @@
 	<f:entry title="${%Description}" field="description">
 		<f:textbox />
 	</f:entry>
+	<f:entry title="${%Always append this description}" field="appendMode">
```

Comment:
To me personally, this seems to emphasise *always* and *append* which might be confising. How about *Append to description instead of resetting it*?